bitnami/kibana/values-production.yaml

```diff
@@ -314,6 +314,15 @@ elasticsearch:
   ## Enable sysctl image for the bundled Elasticsearch
   sysctlImage:
     enabled: false
+  ## Elasticsearch master-eligible node parameters
+  master:
```

Comment:
I wouldn't change it for the production values.

Reply:
Note that Elasticsearch is disabled by default in the production configuration. I've changed it nonetheless.